osmosis-1/osmosis.zone.json

```diff
@@ -1334,6 +1334,14 @@
       "osmosis_main": false,
       "osmosis_frontier": true,
       "osmosis_info": true
+    },
+    {
+      "chain_name": "juno",
+      "base_denom": "cw20:juno1u45shlp0q4gcckvsj06ss4xuvsu0z24a0d0vr9ce6r24pht4e5xq7q995n",
+      "pools": {},
+      "osmosis_main": false,
+      "osmosis_frontier": true,
```

Comment:
osmosis_frontier requires that at least 1 pool is defined

Reply:
How do you create a pool if assets are not available then ?

Reply:
Using CLI. but we can create a staging link of Osmosis with the token if needed.